addon/lib/shooter.js

```diff
@@ -536,8 +536,12 @@ class Shot extends AbstractShot {
       url: url,
       content: body,
       contentType: "application/json",
-      onComplete: watchFunction(function (response) {
+      onComplete: watchFunction((response) => {
         if (response.status >= 200 && response.status < 300) {
+          for (let clipId in response.json.clips) {
+            let clip = this.getClip(clipId);
+            clip.image.url = response.json.clips[clipId].image.url;
```

Comment:
We also have to figure out what this means with respect to whether the clip has changed.  I think this isn't really an edit, but the shot will probably(?) report itself as being dirty as a result of this transformation.

I think the shot object itself should take care of this update.  E.g., `shot.updateClipUrls(response.json)`